### DISCUSSION TITLE
Resnik, Lin similarity ?

### DISCUSSION THREAD

**Discussion by vsraptor** [goodmami/wn#133] [Q&A] [answered]
Any plans on implementing Resnik, Lin similarity ?

**Comment by goodmami** [comment, marked as answer]
Yes, there are plans for Resnik, Lin, and Jiang-Conrath similarity. All three depend on Information Content, which is nearly complete in another branch (see #40). I'll be creating some issues to track those features.

**Comment by goodmami** [comment]
Alright, I've created #122, #123, and #124 to track the three methods so I'll close this now. Please follow the new issues if you're interested in one or more metrics.
